tests/unit/test_credentials.py

```diff
@@ -0,0 +1,85 @@
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+""" Unit tests for the credentials module. """
+
+import unittest
+
+from libcodechecker.libclient import credential_manager
+
+
+class CredentialsTest(unittest.TestCase):
+    """
+    Testing the credentials.
+    """
+
+    def test_get_auth_string(self):
+        """
+        Get authentication string from the credentials.
+        """
+        credentials = {
+            "client_autologin": True,
+            "credentials": {
```

Comment:
Could you please update the documentation (https://github.com/Ericsson/codechecker/blob/master/docs/authentication.md#preconfigured-credentials) with an example where protocol is given?
```js
"https://myserver.com": "serveruser:serverpassphrase",
```